codex-rs/tui/src/history_cell.rs

```diff
@@ -706,16 +711,31 @@ impl SessionHeaderHistoryCell {
         }
     }
 
-    fn format_directory(&self) -> String {
-        if let Some(rel) = relativize_to_home(&self.directory) {
+    fn format_directory(&self, max_width: Option<usize>) -> String {
+        Self::format_directory_inner(&self.directory, max_width)
+    }
+
+    fn format_directory_inner(directory: &Path, max_width: Option<usize>) -> String {
```

Comment:
The fact that this doesn't take some form of `self` suggests it would be more appropriate as a top-level function, but we can live with this.